blocks/research-publications-cards/research-publications-cards.css

```diff
@@ -0,0 +1,158 @@
+.title-button-wrapper{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 50px;
+
+    .title{
+        text-transform: uppercase;
+        font-size: var(--size-28);
+        line-height: 31px;
+
+        p{
+            margin: 0;
+        }
+    }
+    
+    a.title-button{
+        background: var(--primary-blue-80);
+        padding: 15px 20px;
+        border-radius: 50px;
+        font-size: var(--size-16);
```

Comment:
Ideally this should be picked up from font variables. Only use this if the value is not available there.